cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/xl/RDTCP/ns-allinone-3.18.1-rdtcp-queuecontroller/ns-3.18.1/.waf-1.7.11-edc6ccb516c5e3f9b892efc9f53a610f/waflib/Tools/c_config.py", line 459, in run_c_code\n    bld.compile()\n  File "/home/xl/RDTCP/ns-allinone-3.18.1-rdtcp-queuecontroller/ns-3.18.1/.waf-1.7.11-edc6ccb516c5e3f9b892efc9f53a610f/waflib/Build.py", line 188, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 171945484: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-O0\', \'-ggdb\', \'-g3\', \'-Wall\', \'-Werror\', \'-DNS3_ASSERT_ENABLE\', \'-DNS3_LOG_ENABLE\', \'-DINT64X64_USE_CAIRO=1\', \'-DHAVE_SYS_TYPES_H=1\', \'-DHAVE_SYS_STAT_H=1\', \'-DHAVE_DIRENT_H=1\', \'-DHAVE_STDLIB_H=1\', \'-DHAVE_GETENV=1\', \'-DHAVE_SIGNAL_H=1\', \'-DHAVE_PTHREAD_H=1\', \'-DHAVE_STDINT_H=1\', \'-DHAVE_INTTYPES_H=1\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
